devel/cquery/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+PortGroup       waf 1.0
+PortGroup       xcodeversion 1.0
+
+name            cquery
+
+categories      devel
+license         MIT
+maintainers     {wyuenho @wyuenho} openmaintainer
+
+description     C/C++/Objective C Language Server Protocol support
+
+long_description \
+    cquery is a highly-scalable, low-latency language server for \
+    C/C++/Objective-C. It is tested and designed for large code bases like \
+    Chromium. cquery provides accurate and fast semantic analysis without \
+    interrupting workflow. \
+    \
+    cquery implements almost the entire language server protocol and provides some \
+    extra features.
+
+platforms       darwin
+os.version      16.0.0
+homepage        https://github.com/cquery-project/cquery
+
+version         20180302
+
+fetch.type      git
+git.url         https://github.com/cquery-project/cquery
```

Comment:
Why don't you use the github PortGroup?

Reply:
I asked that already. Apparently it doesn't have support for submodules.

Reply:
@pmetzger https://guide.macports.org/#reference.portgroup.github.submodule

Reply:
https://github.com/macports/macports-ports/pull/1989#discussion_r194401624

Reply:
@wyuenho I now see the folded comment, but check the link above.

Reply:
I don't understand what you want me to do. Change pre-configure to post-fetch?

Reply:
@wyuenho See [k6](https://github.com/macports/macports-ports/blob/5768c58be21d9b4997f74314ec2ee7da6887a4ad/net/k6/Portfile). The github PortGroup does "support" submodules, so please use it for clarity.